### ISSUE TITLE
JetEngine - create taxonomy with forms

### ISSUE THREAD

**Issue by joseehilton147** [CrocoBlock/suggestions#2192]
Add possibility to create taxonomy (category, tags, etc) with forms too instead only posts... i think is a must have feature to make front end more simple... for example, if i want to someone create a new category without give him backend access i will only let he create taxonomy trough a form like we already have with posts...

**Comment by dantetesta** [comment]
This function is very useful .... the Piotnet PAFE plugin have this feature and much more ....

I Need make insertions into taxonomy through front forms.

Please add this feature

**Comment by claujohnson** [comment]
I agree. I will have to give access to the backend just because of the lack of this feature. A magnificent job thrown away

**Comment by tylertyza** [comment]
This would be a great feature, as I am in the same position and cannot progress. I think I’ll have to make a separate post type for categories and then assign the posts as children of the category post type.. might get me out for trouble or anyone else running into this issue.

**Comment by EnvioMaik** [comment]
+999 it's been three years, somehow this basic and crucial feature is still missing, even after all the requests on Github. I will have to switch to Piotnet, Fluent Forms or WP Forms now, very sad.

**Comment by aalexandr21** [comment]
was added and released in JetFormBuilder 3.5.2
